From 5864b5078a0187a1a16d4d2ed6f55972511d0a94 Mon Sep 17 00:00:00 2001
From: Jie Luo <jieluo@google.com>
Date: Wed, 27 Nov 2024 14:28:39 -0800
Subject: [PATCH] Revert "Remove deprecated service.py usages from test". For
 29.x only (#19434)

This reverts commit 471ff7727031d7ca17c62f34c38314758142393e.
---
 python/google/protobuf/internal/generator_test.py        | 7 +++++++
 .../google/protobuf/internal/service_reflection_test.py  | 9 ++++-----
 python/google/protobuf/service.py                        | 8 ++++----
 3 files changed, 15 insertions(+), 9 deletions(-)

[ dannf: Only include the changes to service.py, the other don't
  apply to this repository ]
diff --git a/python/google/protobuf/service.py b/python/google/protobuf/service.py
index 8002c040f..38b821791 100644
--- a/python/google/protobuf/service.py
+++ b/python/google/protobuf/service.py
@@ -70,12 +70,12 @@ class Service(object):
     Postconditions:
 
     * "done" will be called when the method is complete.  This may be
-     before CallMethod() returns or it may be at some point in the future.
+      before CallMethod() returns or it may be at some point in the future.
     * If the RPC failed, the response value passed to "done" will be None.
-     Further details about the failure can be found by querying the
-     RpcController.
+      Further details about the failure can be found by querying the
+      RpcController.
     """
-   raise NotImplementedError
+    raise NotImplementedError
 
   def GetRequestClass(self, method_descriptor):
     """Returns the class of the request message for the specified method.
-- 
2.45.2

